src/components/App.jsx

```diff
@@ -1,21 +1,27 @@
+  
 import React, { Component } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import DefaultLayout from '../layouts/default.layout.jsx';
 import MainLayout from '../layouts/main.layout.jsx';
+import ForgotPassword from '../views/password/ForgotPassword';
+import ResetPassword from '../views/password/ResetPassword ';
+import ResetNotification from './password/passwordNotification.jsx';
 
 class App extends Component {
 	render() {
 		return (
 			<div className='app'>
 				<Router>
 					<Switch>
-						<Route path='/' exact component={MainLayout} />
 						<Route path='/auth' component={DefaultLayout} />
+            <Route path='/ForgotPassword' exact component={ForgotPassword} /> 
+            <Route path='/ResetPassword' exact component={ResetPassword} /> 
+            <Route path='/ResetNotification' exact component={ResetNotification } /> 
```

Comment:
naming routes path with while you starting with capital letter I don't think it is a best practice. what do you think naming route paths like forgot-password or any appropriate name instead of ForgotPassword

Reply:
+1 @Jajabenit250